src/AWS.Deploy.Orchestration/Data/AWSResourceQueryer.cs

```diff
@@ -102,7 +106,7 @@ public async Task<List<KeyPairInfo>> ListOfEC2KeyPairs()
             return response.KeyPairs;
         }
 
-        public async Task<string> CreateEC2KeyPair(string keyName, string saveLocation)
+        public async Task<string> CreateEC2KeyPair(string? keyName, string saveLocation)
```

Comment:
https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateKeyPair.html

KeyName is required param, a nullable keyName will result in runtime exception.

Reply:
done